src/styles/main.scss

```diff
@@ -1,3 +1,76 @@
+$size: 100px;
+$gap: 1px;
+$calendar_padding: 10px;
+$calendar_size: $size * 7 + $calendar_padding * 2;
+
+@mixin square($size) {
+  height: $size;
+  width: $size;
+}
+
 body {
   margin: 0;
 }
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  width: $calendar_size;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+
+  $days: (
+    "tue": 0,
+    "wed": 1,
+    "thu": 2,
+    "fri": 3,
+    "sat": 4,
+    "sun": 5,
+    "mon": 6,
+  );
+
+  @each $day, $index in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: $size * $index + $gap * $index;
+    }
+  }
+
+  &_day {
+    @include square($size);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    gap: $gap;
+    border: 1px solid black;
+    background-color: #eee;
+
+    &:hover {
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
Make sure to add transition style under general selector, not the one with :hover - this way transition will work smoothly both ways.